.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: node_js
+node_js:
+- 11.1.0
+before_install:
+- curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.10.1
+- export PATH="$HOME/.yarn/bin:$PATH"
+install:
+- yarn install
+- yarn global add codecov
```

Comment:
Why are you adding codecov as a global here? Can't you do:

```
yarn add codecov
yarn install
```

Reply:
I blindly followed the instructions given by codecov, but yes, it should work too.